packages/react/src/components/TabContent/TabContent.tsx

```diff
@@ -66,9 +87,3 @@ TabContent.propTypes = {
    */
   selected: PropTypes.bool,
 };
-
-TabContent.defaultProps = {
-  selected: false,
-};
```

Comment:
Also not true as the default "falsy" `undefined` serves the same as `false`

Reply:
The rule is `optional bool` instead of `required bool`, which from the means that you can do either of:
- `<Foo />`
- `<Foo bar />`
- `<Foo bar={true} />`: this one should be caught by some linter rule as extraneous, since it's the same as not setting the value
- `<Foo bar={false} />`: This would only be needed if the property was optional and had a fallback to `true`, which is a DX antipattern